gravitee-apim-console-webui/src/management/settings/api-portal-header/migrated/api-portal-header-edit-dialog/api-portal-header-edit-dialog.component.spec.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { MAT_DIALOG_DATA, MatDialogModule, MatDialogRef } from "@angular/material/dialog";
+
+import { ApiPortalHeaderEditDialogComponent } from './api-portal-header-edit-dialog.component';
+
+describe('HeaderDialogComponent', () => {
```

Comment:
I think we can either delete this test because it's covered by the parent component. Or add more complete tests here, like "add new header" or "edit new header". 
but just a test with `expect(component).toBeTruthy()` is not really interesting